src/data/ACTIONS/root/DNC.ts

```diff
@@ -84,7 +84,7 @@ export const DNC = ensureActions({
 		combo: {
 			start: true,
 		},
-		statusesApplied: ['FLOURISHING_SYMMETRY'],
+		statusesApplied: ['SILKEN_SYMMETRY'],
```

Comment:
I'm unclear why you're changing these in the root instead of a layer, this would break for 6.0 patches. Same goes for the rest of this file. Is this just changing the key to use the newer names instead of changing the IDs in the layer?

Reply:
Yes, I changed the key for the old "formerly combined, now just combo-sourced" status the base layer to match the new name for the status, so the base layer needed to use the new key as well. The layers update the name of the old status to match the key, and fix the statusesApplied for the Flourish action to use the new-id-with-the-old-name proc statuses.

Since there was no noticeable change for end users aside from DNC because of the core change, I didn't put in a core changeling along with it, but if we feel that's useful information for users to have I can add one tonight.

Reply:
Checked with Ack, since the core change is transparent to end users other than DNC, we're fine to leave the core changelog alone.

I did switch the base layer definition for the new Flourishing statuses to SHARED.UNKNOWN and moved the real definitions into the 6.1 layer though, now that I know that's a thing!